include/deal.II/lac/dynamic_sparsity_pattern.h

```diff
@@ -342,7 +342,9 @@ class DynamicSparsityPattern : public Subscriptor
                           const IndexSet &rowset = IndexSet());
 
   /**
-   * Create a square SparsityPattern using the index set.
+   * Create a square SparsityPattern using the given index set. The total size
+   * is given by the size of @p indexset and only rows contained in @p
+   * indexset are stored (the @p rowset).
```

Comment:
It might be worth mentioning that `rowset` is a member variable of the class (it was not clear to me what you meant before I read the source).